packages/solid/package.json

```diff
@@ -63,7 +63,7 @@
   },
   "dependencies": {
     "@web3-onboard/common": "^2.3.3",
-    "@web3-onboard/core": "^2.21.0",
+    "@web3-onboard/core": "^2.21.3-alpha.1",
```

Comment:
Can you please bump the package version following the alpha release flag up on line 3 of this file?